cmd/ketch/builder.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+)
+
+const builderCmdHelp = `
+Manage builders.
+`
+
+func newBuilderCmd(cfg config, out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "builder",
+		Short: builderCmdHelp,
+		Long:  builderCmdHelp,
```

Comment:
can we give a little info about what does a builder mean? Users might not be familiar with it that we use buildpacks.

Reply:
I updated the description to include more information on the builder, as well as provided a link to the builder documentation.